docs/user-guide/04-migration-guides.md

```diff
@@ -2,10 +2,29 @@
 This is a list of all recent changes that came with new Antares Simulator features. The main goal of this document is to lower the costs of changing existing interfaces, both GUI and scripts.
 
 ## v9.2.0
-### Adequacy Patch LMR
-Removed following properties from **settings/generaldata.ini**.
-- enable-first-step
-- set-to-null-ntc-between-physical-out-for-first-step
+### Input
```

Comment:
Seems okay to me.

Even though it's not modified in this PR, I have trouble understanding the part concerning the TS-Gen for links.

1- Concerning the ini file, you wrote this

```python

tsgen_direct_XXX,

tsgen_indirect_XXX with XXX in
 ```
but I think the explanation on the `XXX` is missing

2- Also you didn't write the default value for `nominalcapacity` 
3- The text is not correctly formatted as the info about new matrices has bullet points related to the ini file properties which is misleading